docker/Dockerfile

```diff
@@ -73,51 +155,98 @@ RUN strip /usr/local/bin/grepperf \
+
+FROM ubuntu:20.04
+
+RUN apt-get update \
+    && apt-get install -y software-properties-common \
+    && apt-get update && apt-get install -y --no-install-recommends \
+    autoconf \
+    bison \
+    build-essential \
+    ca-certificates \
+    ccache \
+    clang-format-7 \
+    clang-tidy \
+    cron \
+    curl \
+    dnsutils \
+    gawk \
+    gdb \
+    git \
+    htop \
+    iproute2 \
+    lcov \
+    libcurl4-openssl-dev \
+    libsecp256k1-dev \
+    libssl-dev \
+    libtool \
+    libxml2-utils \
+    logrotate \
+    net-tools \
+    ninja-build \
+    nload \
+    ocl-icd-opencl-dev \
+    openssh-client \
+    pkg-config \
+    python3-dev \
+    python3-pip \
+    rsync \
+    rsyslog \
+    tar \
+    trickle \
+    unzip \
+    vim \
+    wget \
+    zip \
+    libexpat1 \
+    && apt-get remove -y cmake python2  && apt-get autoremove -y
 
 # make dirs for scilla and zilliqa
 RUN mkdir -p \
     /scilla/0/bin /scilla/0/src/stdlib \
     /zilliqa/scripts
 
-ARG INSTALL_DIR=/usr/local
-ARG SOURCE_DIR=/root/zilliqa
-
 # pour in scilla binaries
 COPY --from=scilla  /scilla/0/bin2                /scilla/0/bin
 # pour in scilla conntract stdlibs
 COPY --from=scilla  /scilla/0/src/stdlib          /scilla/0/src/stdlib
 # pour in zilliqa scripts
-COPY --from=builder ${SOURCE_DIR}/scripts         /zilliqa/scripts
+COPY --from=builder /zilliqa/scripts         /zilliqa/scripts
 # pour in zilliqa binaries and dynnamic libs
-COPY --from=builder ${INSTALL_DIR}/bin/*     ${INSTALL_DIR}/bin/
-COPY --from=builder ${INSTALL_DIR}/lib/*.so* ${INSTALL_DIR}/lib/
-COPY --from=builder ${INSTALL_DIR}/lib/python3.10 ${INSTALL_DIR}/lib/python3.10
-COPY --from=builder ${SOURCE_DIR}/evm-ds/target/release/evm-ds ${INSTALL_DIR}/bin/
-COPY --from=builder ${SOURCE_DIR}/evm-ds/log4rs.yml ${INSTALL_DIR}/etc/
+COPY --from=builder /usr/local/bin/*     /usr/local/bin/
+COPY --from=builder /usr/local/lib/*.so* /usr/local/lib/
+COPY --from=builder /usr/local/lib/python3.10 /usr/local/lib/python3.10
+COPY --from=builder /zilliqa/evm-ds/target/release/evm-ds /usr/local/bin/
+COPY --from=builder /zilliqa/evm-ds/log4rs.yml /usr/local/etc/
 
 ADD https://github.com/krallin/tini/releases/latest/download/tini /tini
 
-ARG PYTHON3_ENV_PATH=${INSTALL_DIR}
-COPY requirements3.txt ./
+ARG PYTHON3_ENV_PATH=/usr/local
+COPY docker/requirements3.txt ./
 RUN update-alternatives --install /usr/bin/python python ${PYTHON3_ENV_PATH}/bin/python3.10 10 \ 
   && update-alternatives --install /usr/bin/python3 python3 ${PYTHON3_ENV_PATH}/bin/python3.10 10 # set python3 as default instead python3
 
-ENV LD_LIBRARY_PATH=${INSTALL_DIR}/lib
+ENV LD_LIBRARY_PATH=/usr/local/lib
 
 RUN python3 -m pip install --upgrade pip setuptools wheel \
   && python3 -m pip install --no-cache-dir -r requirements3.txt
 
 RUN echo '#!/bin/bash\npython3 -m pip "$@"' > /usr/bin/pip3 \
   && chmod a+x /usr/bin/pip3 \
+  && rm /usr/bin/pip \
```

Comment:
Why is this needed? if this is a remainder from a previous apt package, it's better to remove using apt and not with rm.